### PR TITLE
10 Desing The Application Layout

### PR DESCRIPTION
**Pull request by leonardk1**
#10 
As A developer
I Want To Design and Develop the Application Layout
So That The Development of all other features may proceed

##### Tasks
- [x] Create the Application Layout
- [x] Add the Application Navigations
- [x] Add Bootstrap to the Application for Styling

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review

**Comment by leonardk1** [comment]
@godfreym please review

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class DashboardController < ApplicationController
```
Comment (godfreym) [inline app/controllers/dashboard_controller.rb]: @leonardk1 controllers are named using plural convention so i think this should be DashboardsController

File: app/views/dashboard/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<div id="page-wrapper">
```
Comment (godfreym) [inline app/views/dashboard/index.html.erb]: @leonardk1 `app/views/dashboards/...` note the plural

File: app/controllers/dashboard_controller.rb (hunk 1 above)
Comment (leonardk1) [inline app/controllers/dashboard_controller.rb]: @godfreym That is True but sometimes there are exceptions, like in this case the dashboard is one, we don't have dashboards but a dashboard

we need somethink like `https://dashboard`

Comment (godfreym) [inline app/controllers/dashboard_controller.rb]: @leonardk1 OK though i am not sure about that, so the convention rule sometimes can not be applicable?

File: app/views/dashboard/index.html.erb (hunk 2 above)
Comment (leonardk1) [inline app/views/dashboard/index.html.erb]: @godfreym I believe this is what I just explained up there

**Comment by godfreym** [comment]
@leonardk1 Apart from my convention comment, All looks Good :+1:

**Comment by leonardk1** [comment]
@godfreym thanks for the review
@arafatm please review

**Comment by arafatm** [comment]
:+1:

**Pull request merged**